app/components/cookies_banner_component.rb

```diff
@@ -0,0 +1,21 @@
+class CookiesBannerComponent < GovukComponent::Base
```

Comment:
Is this file just the deleted `app/components/ab_test/cookies_banner_component.rb` but moved?

Reply:
yes and a few stuctural bits added for default component tests (html_attrs for e.g)

Reply:
I thought most IDEs were capable of recording when files are renamed/moved. It makes git history easier to understand. I think you can also use `git mv hello.html directory` instead of `mv hello.html directory` to make git history show that it's a moved file rather than two files that were deleted/added.

No worries for this PR but it is a nice-to-have for the future.

Reply:
you are absolutely right and I even mentioned this not do long ago in an eng workshop. It did cross my mind just after I created Pr but I came to conclusion the original only had 2 updates for abyest and in fact weren't losing isting anything useful in term of history